content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -233,6 +236,20 @@ repo_gpgcheck=1
 gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
 EOF
 
+#For PPC 
+
+cat <<EOF > /etc/yum.repos.d/kubernetes.repo
+[kubernetes]
+name=Kubernetes
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-ppc64le
+enabled=1
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+EOF
```

Comment:
hi, 
we don't have plans to enumerate all the supported architectures in the kubeadm pages.

instead of adding a separate example for PPC, i'd do the following:
> baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-ARCH

add a note on top saying:
> replace ARCH with the preferred architecture, for example `x86_64`.

...but i would like to enable this for all the supported distro families and not only Redhat*
so my question here is - how to enable this for the Debian docs too?

/hold 
/sig cluster-lifecycle